stack/stack.py

```diff
@@ -3,10 +3,7 @@ def __init__(self):
         self._list = []
 
     def __str__(self):
-        return str(self._list)
-
-    def __len__(self):
-        return len(self._list)
+        raise Exception("Not allowed to view stack contents")	
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/stack/stack.py
+++ b/stack/stack.py
@@ -3,7 +3,7 @@
         self._list = []
 
     def __str__(self):
-        raise Exception("Not allowed to view stack contents")	
+        raise Exception("Not allowed to view stack contents")
 
     def push(self, item):
         self._list.append(item)
```